lib/files/schema/simulationparameter.json

```diff
@@ -0,0 +1,313 @@
+            "default": false,
+            "type": "boolean"
+          },
+          "outputs": {
+            "title": "Outputs",
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "summary_reports": {
+            "title": "Summary_Reports",
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          }
+        },
+        "required": [
+          "type"
+        ]
+      },
+      "Date": {
+        "title": "Date",
+        "description": "Date.",
+        "type": "object",
+        "properties": {
+          "is_leap_year": {
+            "title": "Is_Leap_Year",
+            "description": "A boolean to indicate if datetime is for a leap year. Default is `false`.",
+            "default": false,
+            "type": "boolean"
+          },
+          "month": {
+            "title": "Month",
+            "description": "A value for month between `1`-`12`. Default is `1`.",
+            "default": 1,
+            "minimum": 1,
+            "maximum": 12,
+            "type": "integer"
+          },
+          "day": {
+            "title": "Day",
+            "description": "A value for day between `1`-`31`. Default is `1`.",
+            "default": 1,
+            "minimum": 1,
+            "maximum": 31,
+            "type": "integer"
+          }
+        }
+      },
+      "DaylightSavingTime": {
+        "title": "DaylightSavingTime",
+        "type": "object",
+        "properties": {
+          "type": {
+            "title": "Type",
+            "enum": [
+              "DaylightSavingTime"
+            ]
+          },
+          "start_date": {
+            "$ref": "#/definitions/Date"
+          },
+          "end_date": {
+            "$ref": "#/definitions/Date"
+          }
+        },
+        "required": [
+          "type",
+          "start_date",
+          "end_date"
+        ]
+      },
+      "RunPeriod": {
+        "title": "RunPeriod",
+        "description": "Used to describe the time period over which to run the simulation.",
+        "type": "object",
+        "properties": {
+          "type": {
+            "title": "Type",
+            "enum": [
+              "RunPeriod"
+            ]
+          },
+          "start_date": {
+            "$ref": "#/definitions/Date"
+          },
+          "end_date": {
+            "$ref": "#/definitions/Date"
+          },
+          "start_day_of_week": {
+            "title": "Start_Day_Of_Week",
+            "default": "Sunday",
+            "enum": [
+              "Sunday",
+              "Monday",
+              "Tuesday",
+              "Wednesday",
+              "Thursday",
+              "Friday",
+              "Saturday"
+            ],
+            "type": "string"
+          },
+          "holidays": {
+            "title": "Holidays",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/Date"
```

Comment:
OpenStudio currently does not support holidays.

Reply:
I must have forgotten that we have holidays implemented in honeybee-legacy through a hack. Did we also not implement `apply_holiday` on the `ScheduleRuleset` schema, then? This really seems like a critical feature that the OpenStudio SDK should support since it is really not possible to simulate a lot of common building programs like religious centers or assembly halls without holidays.  I would say that this is probably the highest priority for any task 3 hours that remain and, if we don't have any hours left here, I guess we'll have to write an EnergyPlus measure to do this.

Reply:
@chriswmackey , I did not implement this in the measure for `ScheduleRuleset`, I do have `apply_holiday` in the schema. Joe had about 30 hours out of the 80 for task 3. He was using core openstudio development to charge his time as well. 
I can ask him how  much time it would take for him to holidays in the OpenStudio SDK.

Reply:
That would be great if some of these features are address-able within the 30 hours.  I am also coming to realize that I'll likely only need 5 of the 50 hours that are budgeted to Matt for the standards gem help.  So it seems worthwhile to send an email to Kristin to see if we might be able to reallocate some of those original standards gem hours to addressing these issues.

Reply:
I spoke to Joe, he has used up the hours on task 3. Matt and Marley collectively had 40 hours allocated. I am not sure aout Joe's availability but the remained of those could be reallocated. We can schedule another call to discuss this,

Reply:
What did Joe use the hours to address? I haven't had any contact with him and I actually didn't know that he was taking over Task 3 from Dan until you mentioned it here. The last work I remember happening on Task 3 was Dan adding in the `setCalculationMethod` to the `ShadowCalculation` object, though I didn't know if Dan had used up all of the Task 3 hours there.  A call to sort all of this out would be helpful.